parlai/core/metrics.py

```diff
@@ -361,7 +361,7 @@ def update(self, observation, labels):
                 for i in range(3):
                     weights = [1 / (i + 1) for _ in range(i + 1)]
                     bleu_scores[f'bleu-{i + 1}'] = _bleu(prediction, labels, weights)
-            if 'rouge-L' in self.metrics_list:
+            if 'rouge-L' in self._print_metrics_list:
```

Comment:
... where did this come from?

Reply:
From noticing the test wasn’t working as intended